charts/pmm/templates/ingress.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.ingress.enabled -}}
+{{- $serviceName := .Values.service.name -}}
+{{- $ingressPathType := .Values.ingress.pathType -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "pmm.fullname" . }}
+  labels:
+    {{- include "pmm.labels" . | nindent 4 }}
+  annotations:
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
```

Comment:
to enable ppl to use different ingress controllers, please also parametrize this one. Same as in `helm create` defaul templates:
```
  {{- with .Values.ingress.annotations }}
  annotations:
    {{- toYaml . | nindent 4 }}
  {{- end }}
```

or another example here: 
https://github.com/prometheus-community/helm-charts/blob/8f0b863eab3eb604216ce4084608d1b6a7f8e879/charts/kube-prometheus-stack/templates/prometheus/ingress.yaml

Reply:
implemented with values example for those needing a hint. (https://github.com/percona/percona-helm-charts/pull/142/commits/86a84ecbdb71b28958a5d325d4037645040c598b)